src/parser/HLSTree.cpp

```diff
@@ -23,6 +23,7 @@
 #include <algorithm>
 #include <optional>
 #include <sstream>
+#include <decrypters/Helpers.h>
```

Comment:
and moved above by sorting it
```suggestion
#include "decrypters/Helpers.h"
```